src/modules/chat_custom_timeouts/index.js

```diff
@@ -29,15 +30,30 @@ const ACTION_TYPES = {
 let action;
 let user;
 
+function setReason(type) {
+    if (settings.get('customTimeoutReasons')) {
+        const reason = prompt(`Enter ${type === ACTION_TYPES.BAN ? 'ban' : 'timeout'} reason: (leave blank for none)`);
```

Comment:
i would inject `${type}` directly .. and swap the timeout action type to `timeout` instead of `time`